### DISCUSSION TITLE
Ré-écrire les aides pour avoir un meilleur aperçu, condensé des infos ?

### DISCUSSION THREAD

**Discussion by lauriechap** [betagouv/ellesfontlaphoto#243] [Sujets traités ou résolus] [answered]
Les dernières aides ajoutées ont des aperçus très longs, difficile à lire. Ne faudrait il pas les adapter comme les autres à un format plus simplifié ?
<img width="1345" alt="Capture d’écran 2022-03-14 à 17 03 14" src="https://user-images.githubusercontent.com/36261410/158212406-587ff193-32fb-4c74-9d05-da991e6f7c04.png">

**Comment by wittsarah** [comment]
Hello Laurie ! Bien vu ! je viens de les modifier, c'est plus clair comme ça normalement :) 

<img width="1548" alt="Capture d’écran 2022-03-15 à 16 27 13" src="https://user-images.githubusercontent.com/101105157/158412714-8a6c1dee-65fc-484f-9615-b7de51a857a1.png">

**Comment by lauriechap** [comment, marked as answer]
Oh génial merci @wittsarah ! c'était pour en discuter mais top que tu t'en sois saisie si rapidement :)
